documents/Project_idea_document.md

```diff
@@ -45,6 +45,47 @@ The possibility to sell the ideas to companies was mentioned. I prefer the idea
 Also, the idea of ads was shot down by the professor, but I believe that if you can show ads that are relevant and non-intrusive it is not a bad idea. For example, there is an popular idea to make a virtual pet dragon app. You can have a web-crawler search for a similar app that is already available (similar but most likely not exactly the same) and offer the makers of the app some adspace on the site. The ad would be relevant to what the user is looking at. The idea on the site can still be elaborated on and might turn into something quite different.
 
 ####Requirements and Specifications
+
+#####a)	User Characteristics
+The main users that the application is created for are both experienced and non-experienced users; it can be used by average users that have ideas that they would like to be made but do not have the required time/skills/money for it. The general users that the application addresses are listed below:
+* 	Non-experienced User:	Average user with little or no expertise. Uses the system to share ideas and/or vote on ideas.
+* Experienced User:	User with expertise in the field, who uses the system to take the ideas to the implementation level. They can be in-house experts who define the chosen ideas. They can also vote on ideas.
+* Investors/Companies:	Users who are looking for new ideas to get implemented and gain profit by funding and/or providing resources for it. They can also vote on ideas that they would like to get implemented.	
+
+#####b)	Functional Requirements
+The application functions in two phases; one for sharing and voting ideas and another for working on the implementation of the ideas. The main functional requirements in these phases are as follows:
+
+Phase 1:
+*	The system allows users to sign up and create a user profile.
+*	The system must allow users to submit ideas for mobile applications, accompanied by a short text describing the idea and an optional picture.
+*	The system must put the idea along with its description through a sanitation check to eliminate submissions that are inappropriate according to specified rules. Sanitation check is performed by CrowdFlower.
```

Comment:
I believe we also decided that the sanitation check includes and IR component that finds similar ideas. The similar ideas found by the IR system are also checked with human computation